openmetadata-service/src/main/java/org/openmetadata/service/resources/services/security/SecurityServiceResource.java

```diff
@@ -0,0 +1,741 @@
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.json.JsonPatch;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.DefaultValue;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.PATCH;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.PUT;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.SecurityContext;
+import jakarta.ws.rs.core.UriInfo;
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.openmetadata.schema.api.data.RestoreEntity;
+import org.openmetadata.schema.api.services.CreateSecurityService;
+import org.openmetadata.schema.entity.services.SecurityService;
+import org.openmetadata.schema.entity.services.ServiceType;
+import org.openmetadata.schema.entity.services.connections.TestConnectionResult;
+import org.openmetadata.schema.type.ChangeEvent;
+import org.openmetadata.schema.type.EntityHistory;
+import org.openmetadata.schema.type.Include;
+import org.openmetadata.schema.type.MetadataOperation;
+import org.openmetadata.schema.type.SecurityConnection;
+import org.openmetadata.schema.type.csv.CsvImportResult;
+import org.openmetadata.schema.utils.JsonUtils;
+import org.openmetadata.service.Entity;
+import org.openmetadata.service.jdbi3.SecurityServiceRepository;
+import org.openmetadata.service.limits.Limits;
+import org.openmetadata.service.resources.Collection;
+import org.openmetadata.service.resources.services.ServiceEntityResource;
+import org.openmetadata.service.security.Authorizer;
+import org.openmetadata.service.security.policyevaluator.OperationContext;
+import org.openmetadata.service.util.CSVExportResponse;
+import org.openmetadata.service.util.ResultList;
+
+@Slf4j
+@Path("/v1/services/securityServices")
+@Tag(
+    name = "Security Services",
+    description = "APIs related to Security Service entities, such as Apache Ranger.")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Collection(name = "securityServices")
+public class SecurityServiceResource
+    extends ServiceEntityResource<SecurityService, SecurityServiceRepository, SecurityConnection> {
+  private final SecurityServiceMapper mapper = new SecurityServiceMapper();
+  public static final String COLLECTION_PATH = "v1/services/securityServices/";
+  public static final String FIELDS = "owners,tags,domain,followers";
```

Comment:
- in `main` should be `domains`
- in 1.8.8 should be `domain`

Let's merge in `main` with the multi domains, otherwise tests will fail. then cherry-pick making the proper adjustements for 1.8.8